docs/README.md

```diff
@@ -66,8 +66,9 @@ The agent is compatible with Kubernetes clusters running `v1.9` or greater. It c
 
 
 ```
-kubectl create -f https://raw.githubusercontent.com/logdna/logdna-agent-v2/master/k8s/logdna-agent.yaml
+kubectl create -f https://raw.githubusercontent.com/logdna/logdna-agent-v2/master/k8s/logdna-agent-ns.yaml
 kubectl create secret generic logdna-agent-key -n logdna-agent --from-literal=logdna-agent-key=<YOUR LOGDNA INGESTION KEY>
```

Comment:
is `-n` in this command line an option for `namespace`? Like the shortened version of `--namespace` and basically, for creating and adding a `secret` into `logdna-agent` namespace, right?

Reply:
Yup, `--namespace` is the long form for `-n`.